packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2883,6 +2885,99 @@ module.exports = function (log, config, bounces) {
     });
   };
 
+  Mailer.prototype.subscriptionFirstInvoiceDiscountEmail = async function (
+    message
+  ) {
+    const {
+      email,
+      uid,
+      productId,
+      planId,
+      planEmailIconURL,
+      productName,
+      invoiceNumber,
+      invoiceDate,
+      invoiceLink,
+      invoiceTotalInCents,
+      invoiceTotalCurrency,
+      subtotal,
+      discountAmount,
+      payment_provider,
+      cardType,
+      lastFour,
+      nextInvoiceDate,
+    } = message;
+
+    const enabled = config.subscriptions.transactionalEmails.enabled;
+    log.trace('mailer.subscriptionFirstInvoiceDiscount', {
+      enabled,
+      email,
+      productId,
+      uid,
+    });
+    if (!enabled) {
+      return;
+    }
+
+    const query = { plan_id: planId, product_id: productId, uid };
+    const template = 'subscriptionFirstInvoiceDiscount';
+    const translator = this.translator(message.acceptLanguage);
+
+    const links = this._generateLinks(null, message, query, template);
+    const headers = {};
+    const translatorParams = { productName };
+    const subject = translator.gettext('%(productName)s payment confirmed');
+
+    return this.send({
+      ...message,
+      headers,
+      layout: 'subscription',
+      subject,
+      template,
+      templateValues: {
+        ...links,
+        ...translatorParams,
+        uid,
+        email,
+        invoiceDateOnly: this._constructLocalDateString(
+          message.timeZone,
+          message.acceptLanguage,
+          invoiceDate
+        ),
+        nextInvoiceDateOnly: this._constructLocalDateString(
+          message.timeZone,
+          message.acceptLanguage,
+          nextInvoiceDate
+        ),
+        icon: planEmailIconURL,
+        product: productName,
+        subject: translator.format(subject, translatorParams),
+        invoiceLink,
+        invoiceNumber,
+        invoiceDate,
+        invoiceTotal: this._getLocalizedCurrencyString(
+          invoiceTotalInCents,
+          invoiceTotalCurrency,
+          message.acceptLanguage
+        ),
+        subtotal: this._getLocalizedCurrencyString(
+          subtotal,
+          invoiceTotalCurrency,
+          message.acceptLanguage
+        ),
+        discountAmount: `-${this._getLocalizedCurrencyString(
```

Comment:
I don't think `-` to the left of the number is universal.

Reply:
I've added it  back into the translations as test. Still doesn't feel quite right. I'll check with the l10n team tomorrow what's best for this.